xml/System.Text.Json.Serialization/JsonExtensionDataAttribute.xml

```diff
@@ -19,8 +19,24 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>When placed on a property of type <see cref="T:System.Collections.Generic.IDictionary`2" />, provides that any properties that do not have a matching member are added to that dictionary during deserialization and written during serialization.</summary>
```

Comment:
Should we specify the generic parameters are TKey, TValue (since we refer to them in the remarks), or is it will understood that IDictionary'2 is `IDictionary<TKey,TValue>`?

https://github.com/dotnet/corefx/blob/8e2f9a765f6533d17e711365ce2a3064b8d1dfc6/src/System.Text.Json/src/System/Text/Json/Serialization/JsonExtensionDataAttribute.cs#L8

Reply:
This question has arised in [other PRs](https://github.com/dotnet/dotnet-api-docs/pull/2359/files) that clearly specify `{something}` at the end of the DocId. It seems that triple slash comments with such elements do not get properly converted to the xml files in artifacts/bin. So when I automatically port them, they look like in this comment you mentioned.

I couldn't find an easy fix, so I opted for something that looks like:

> `An <see cref="T:System.Collections.Generic.IDictionary`2" /> of <typeparamref name="TKey" /> and <typeparamref name="TValue" />`

But I haven't heard back from the Docs team for confirmation.

There may be another solution: to parse the html entities for the curly brackets. I don't know if that will build successfully in the Docs repo.

Reply:
More info: I learned about this limitation in triple slash comments from [this StackOverflow question](https://stackoverflow.com/questions/532166/how-to-reference-generic-classes-and-methods-in-xml-documentation/41208166#41208166).

Reply:
I'm not quite sure that I understand the issue. At build time, Dictionary`2 renders as:
![image](https://user-images.githubusercontent.com/10886961/61498957-3c0b7980-a97a-11e9-856a-9294b320e33f.png)

Reply:
In that case, all's good. Thanks @rpetrusha.

@carlossanlop - in our case here, we are not passing a custom (or specific TKey, TValue) type in the comments, so the issue you mentioned isn't relevant here.

If, for example, if the xml comment specified something like `IDictionary{string,object}`, we would need to do something different to  display that.

Reply:
Resolving since I misunderstood the problem and it all looks good.